vcstool/commands/import_.py

```diff
@@ -33,12 +38,20 @@ def __init__(self, args, url, version=None, recursive=False):
         self.recursive = recursive
 
 
+def file_or_url_type(x):
+    try:
+        return urlopen(x)
```

Comment:
In case the URL parsing raises an exception a stacktrace is shown which isn't user friendly. Please catch the relevant exceptions and print a single line error message.